lib/inets/src/http_server/httpd_util.erl

```diff
@@ -546,6 +605,10 @@ from left-to-right on the hunt for longest possible `Path` being a file or a
 directory. Everything after the longest possible `Path`, isolated with a `/`, is
 regarded as `PathInfo`
 """.
+-spec split_path(RequestLine) -> {Path,QueryStringOrPathInfo} when
```

Comment:
Should this not be an URI  (as the variable) or maybe URIString, a RequestLine would be the whole "GET URI Version" thing!